tests/reporter/kubernetes.go

```diff
@@ -918,3 +1024,157 @@ func writeStringToFile(filePath string, data string) error {
 	_, err = f.WriteString(data)
 	return err
 }
+
+func getVmiType(vmi v12.VirtualMachineInstance) (string, error) {
+	for _, volume := range vmi.Spec.Volumes {
+		if volume.VolumeSource.ContainerDisk == nil {
+			continue
```

Comment:
I think it would help with debugging if you will return a different error here
```
return "", fmt.Errorf("containerDisk is nil for vmi  %s", vmi.ObjectMeta.Name)
```

Reply:
Some volumes don't have ContainerDisk and its ok, there might be couple volumes, we don't want to print all the non ContainerDisk ones, but we do `continue`